.gitignore

```diff
@@ -20,3 +20,4 @@
 /node_modules/
 /.tmp
 .DS_Store
+/.idea
```

Comment:
You shouldn't add this here but globally on your system. If we did here, we would have to add every IDE.